src/http.js

```diff
@@ -340,19 +385,51 @@ function decodeContent(html, contentType) {
  * @param {Object} options
  * @return {Promise<Object>} response, body
  */
-function customRequest(method, requestURL, options) {
+
+var getThrottledRequestForURL = (function() {
+    var i = 0;
+    return function(requestURL) {
+        console.log("REQUEST URL: " + requestURL);
+        console.log("Request Number: " + ++i);
+        //return throttledRequestsForURLs[i % 100];
+        let domain = getDomainForURL(requestURL);
+        // Customized request object exists
+        if (throttledRequestsForURLs[domain])
+            return throttledRequestsForURLs[domain];
+        // Create new customized request object if custom value exists
+        var requestIntervalForDomain = getRequestIntervalForDomain(domain);
+        if (requestIntervalForDomain) {
+            throttledRequestsForURLs[domain] = require('throttled-request')(request);
+            throttledRequestsForURLs[domain].configure({
+               requests: 1,
+               milliseconds: requestIntervalForDomain
+            });
+            throttledRequestsForURLs[domain].on('request', function () {
+                 console.log('Making a request (throttledRequest' + domain + '). Elapsed time: %d ms', Date.now() - startedAt);
+            });
+            console.log("CONFIG: " + i + " - " +  JSON.stringify(throttledRequestsForURLs[domain].get('config')));
+            return throttledRequestsForURLs[domain];
+       }
+       // Return default object
+       return throttledRequestsForURLs['default'];
+    };
+})();
+
+
+function customThrottledRequest(method, requestURL, options) {
+    var throttledRequestForURL = getThrottledRequestForURL(requestURL);
```

Comment:
Strange indentation compared to the next line

Reply:
Sorry, cannot see the indentation problem.